cfgov/unprocessed/css/enhancements/base.less

```diff
@@ -13,3 +13,13 @@ nav ol ol {
 }
 
 // END enhancement for normalize-legacy-addon
+
+h1,
+.h1 {
+    .respond-to-max( @bp-xs-max, {
```

Comment:
Should `.respond-to-max( …` enclose the `h1, .h1 {`?

Reply:
I agree with Ans that this would be cleaner. I'd also simplify the selector to

```less
.eyebrow + h1,
.eyebrow + .h1 {
```